Python.md

```diff
@@ -40,7 +85,37 @@ result = connection.execute(sqlalchemy.text("""
            show_gravatar
       FROM "user"
      WHERE musicbrainz_id IN :musicbrainz_usernames
-"""), {
-    'musicbrainz_usernames': tuple(usernames),
-})
+""")
+result = connection.execute(query, {'musicbrainz_usernames': tuple(usernames)})
 ```
+
+It is preferred to assign the query to a separate variable and not include it inside the
+`execute` call.
+
+TODO: Projects which don't use sqlalchemy
+
+### Tests
+
+You should write tests for all new methods that you write. If you update a method, make sure that
+tests for that method still pass, and write/remove tests as necessary for the change in functionality.
+
+Tests for a module (`package/module.py`) go in a `test` subdirectory - `package/test/test_module.py`
+
```

Comment:
It might be a good idea to add a subsection "Database access" for the content below.